inspirehep/manage.py

```diff
@@ -210,3 +209,19 @@ def recreate_index(name, mapping, rebuild=False, delete_old=True):
         es.indices.create(index=name + "_v1", body=mapping)
         es.indices.put_alias(index=name + "_v1", name=name)
     return True
+
+
+@manager.command
+def invalidate_cache(cache_name):
+    """Invalidate cache of certain type e.g. brief, citation etc."""
+    from inspirehep.ext.cache_manager import cache_manager
+    template_name_to_function = {
+        'all': cache_manager.invalidator.invalidate_all_cached_views,
+        'citation': cache_manager.invalidator.invalidate_all_citation_views,
+        'brief': cache_manager.invalidator.invalidate_all_brief_search_views,
+        'references': cache_manager.invalidator.invalidate_all_references_views,
+        'citations': cache_manager.invalidator.invalidate_all_citations_views,
+        'reference': cache_manager.invalidator.invalidate_all_reference_views
```

Comment:
Singular `reference ` is a single entry in the plural `references` field. Just realized though that it can be reimplemented so we could use singular `citation` for that and get rid of singular `reference`.